setup.py

```diff
@@ -48,6 +51,20 @@
 else:
     assert False, sys.platform + " not supported"
 
+if IS_LIN:
+    DPCPP_ROOT = os.environ["ONEAPI_ROOT"] + "/compiler/latest/linux"
+    os.environ["CC"] = DPCPP_ROOT + "/bin/clang"
+    os.environ["CXX"] = DPCPP_ROOT + "/bin/dpcpp"
+    os.environ["DPPL_SYCL_INTERFACE_LIBDIR"] = "dpctl"
+    os.environ["DPPL_SYCL_INTERFACE_INCLDIR"] = "dpctl/include"
+    os.environ["CFLAGS"] = "-fPIC"
+
+elif IS_WIN:
+    os.environ["CC"] = "clang-cl.exe"
+    os.environ["CXX"] = "dpcpp.exe"
```

Comment:
I think next step (in another PR) should be to try allow Python to build Cython modules with default compiler (not DPC++).